server/src/test/resources/csars/yaml/valid/lamp-csar_Input/template.yaml

```diff
@@ -0,0 +1,82 @@
+tosca_definitions_version: tosca_simple_yaml_1_0
+description: Template for deploying a LAMP stack.
+metadata:
+  template_name: lamp-stack-template
+  template_author: stupro-toscana
+  template_version: 1.0
+
+
+topology_template:
+  inputs:
+    database_name:
+      type: string
+    database_port:
+      type: integer
+    database_password:
+      type: string
+
+  node_templates:
+    my_app:
+      type: tosca.nodes.WebApplication
+      requirements:
+        - host: apache_web_server
+        - database_endpoint: my_db
+      interfaces:
+        Standard:
+          inputs: db_host: { get_attribute: [ server, private_address ] }
+          create: my_app/install_php.sh
+          start:
+            implementation:
+              primary: my_app/start_myphpapp.sh
+              dependencies:
+                - myphpapp.php
+                - mysql-credentials.php
+
+    apache_web_server:
+      type: tosca.nodes.WebServer.Apache
+      requirements:
+        - host: server
+
+    my_db:
+      type: tosca.nodes.Database.MySQL
+      properties:
+        name: { get_input: database_name }
+        password: { get_input: database_password }
+        port: { get_input: database_port }
+      capabilities:
+        database_endpoint:
+          properties:
+            port: { get_property: [ SELF, port ] }
+      requirements:
+        - host: mysql_dbms
+
+    mysql_dbms:
+      type: tosca.nodes.DBMS.MySQL
+      properties:
+          root_password: { get_input: database_password }
+          port: { get_input: database_port }
+        requirements:
+          - host: server
+        interfaces:
+          Standard:
+            inputs:
+              db_root_password: { get_property: [ SELF, root_password ] }
+            configure: mysql_dbms/mysql_dbms_configure.sh
+
+    server:
+      type: tosca.nodes.Compute
+      capabilities:
+        host:
+          properties:
+            num_cpus: 1
+            disk_size: 25 GB
+            mem_size: 2048 MB
+        os:
+          properties:
+            type: linux
+            distribution: ubuntu
+            version: 16.04
+    outputs:
+      endpoint:
+        description: endpoint of the php-app
+          value: { get_attribute: [ server, public_address ] }
```

Comment:
Technically this is not the endpoint, but the public address (endpoint is public address + port).
Maybe somehow expose some attributes (port, url..) of app_endpoint (EndpointCapability) of Webapplication.